src/components/inputs/money-input/money-input.js

```diff
@@ -333,14 +338,16 @@ export default class MoneyInput extends React.Component {
     // value to a number, as we want to keep a string!
     // The fake event does not contain the input type information, so Formik
     // will not convert the value to a number.
-    const fakeEvent = {
-      persist: () => {},
-      target: {
-        name: event.target.name,
-        value: event.target.value,
-      },
-    };
-    this.props.onChange(fakeEvent);
+    if (isNumberish(event.target.value)) {
+      const fakeEvent = {
+        persist: () => {},
+        target: {
+          name: event.target.name,
+          value: event.target.value,
+        },
+      };
+      this.props.onChange(fakeEvent);
```

Comment:
Now that we're not using an `<input type="number" />` we avoid the problems formik introduced by trying to auto-convert the value to a number. I think we can just do

```js
if (isNumberish(event.target.value)) this.props.onChange(event)
```

and delete the comment as well!